carp.common/src/commonMain/kotlin/dk/cachet/carp/common/data/CarpDataTypes.kt

```diff
@@ -46,5 +46,11 @@ object CarpDataTypes : EnumObjectList<DataType>()
      * Heart rate. Measures the number of heart contractions (beats) per minute.
      */
     val HEARTRATE = add( DataType.fromFullyQualifiedName( HEARTRATE_TYPE_NAME ) )
+
+    internal const val SENSORCONTACT_TYPE_NAME = "$CARP_NAMESPACE.sensorcontact"
+    /**
+     * Sensor contact. An indicator of proper skin contact. Typically found in HR chest straps.
+     */
+    val SENSORCONTACT = add( DataType.fromFullyQualifiedName(SENSORCONTACT_TYPE_NAME) )
```

Comment:
Added separate data type for sensor contact. Chose generalized form as more sensors than just BLE hr might formally or informally support this. Considered to just add the contact flag to the hr data type, but that did not seem right because we should avoid having nullable data in there imho.